.travis.yml

```diff
@@ -140,4 +146,25 @@ jobs:
         - travis_wait make test-e2e-images
         - odo logout
 
+    # Run devfile integration test on Kubernetes cluster    
+    - <<: *base-test
+      stage: test
+      name: "devfile catalog, create, push, watch command integration tests"
```

Comment:
Sorry, i missed this one.
ATM you are testing only ```devfle catalog```, so instead of all test file name can you pleas update the name to ```devfile catalog command integration tests on Kubernetes cluster```

Reply:
> Sorry, i missed this one.
> ATM you are testing only `devfle catalog`, so instead of all test file name can you pleas update the name to `devfile catalog command integration tests on Kubernetes cluster`

I think name ```devfile catalog command integration tests``` is more specific as the job comment itself says that it is running on kubernetes cluster.

Reply:
This is the job display name in the build dash board. 

<img width="1089" alt="Screen Shot 2020-04-15 at 11 50 03 AM" src="https://user-images.githubusercontent.com/19851115/79304646-73aa5080-7f0f-11ea-82bb-a9e1ad2347af.png">

So the name helps us to recognize what job is running which test.

Reply:
@amitkrout Yes, but from user point of view specifying kubernetes cluster in the name will be useful. I did not think from this side earlier. Thanks for pointing it out +1